code/game/area/areas.dm

```diff
@@ -28,8 +28,6 @@
 	var/unique = TRUE
 
 	var/has_gravity = 1
-	var/area/master // master area used for power calcluations
-								// (original area before splitting due to sd_DAL)
 	var/list/related // the other areas of the same type as this
```

Comment:
`related` is pretty much the counterpart to `master`, believe it can be stripped away with the rest